docs/courses/csintro3/orientation/math.md

```diff
@@ -1,101 +1,171 @@
 # Math
 
-table with math blocks -> equivalent
+When transitioning into JavaScript, we see that doing calculations and writing math formulas becomes a lot easier.
```

Comment:
For a more consistent voice, generally avoid we / etc: for example, something like

In Blocks, mathematical calculations and formulas quickly became complex and hard to follow. In JavaScript, the same formulas can be written in a more compact, easy to read format.